src/chains/ethereum/ethereum/src/miner/executables.ts

```diff
@@ -1,7 +1,42 @@
 import { TypedTransaction } from "@ganache/ethereum-transaction";
 import { Heap } from "@ganache/utils";
+import { byNonce } from "../transaction-pool";
 
-export type Executables = {
-  inProgress: Set<TypedTransaction>;
-  pending: Map<string, Heap<TypedTransaction>>;
-};
+export class Executables {
+  public inProgress: Set<TypedTransaction> = new Set();
+  public pending: Map<string, Heap<TypedTransaction>> = new Map();
```

Comment:
I assume that the key on the map is the `from` address. Would it be worthwhile naming the property `pendingBySender` or in some other way indicating this?